discourse_migration/userguide.md

```diff
@@ -0,0 +1,152 @@
+| llvm-branch-commits   | no migration at the moment |
+| llvm-bugs             | no migration at the moment |
+| llvm-commits          | no migration at the moment |
+| llvm-dev              | lvm-project/llvm |
+| llvm-devmeeting       | community/devmtg |
+| llvm-foundation       | **TODO** |
+| Mlir-commits          | no migration at the moment |
+| Openmp-commits        | no migration at the moment |
+| Openmp-dev            | lvm-project/openmp |
+| Parallel_libs-commits | no migration at the moment |
+| Parallel_libs-dev     | **TODO** |
+| Release-testers       | **TODO** |
+| Test-list             | **TODO** |
+| vmkit-commits         | **TODO** |
+| WiCT                  | **TODO** |
+| www-scripts           | **TODO** |
+
+**TODO:** add discourse links once the list is finalized
+
+## FAQ
+
+### I don't want to use a web UI
+
+You can do most of the communication with your email client (see section on
+Setting up email interactions above). You only need to set up your account once
+and then configure which categories you want to subscribe to.
+
+### How do I send a private message?
+
+On the mailing list you have the opportunity to reply only to the sender of
+the email, not to the entire list. That is not supported when replying via
+email on Discourse. However you can send someone a private message via the
+Web UI: Click on the user's name above a post and then on `Message`.
+
+Also Discourse does not expose users' email addresses , so your private
+replies have to go through their platform (unless you happen to know the
+email address of the user.)
+
+### How can my script/tool send automatic messages?**
+
+In case you want to [create a new
+post/topic](https://docs.discourse.org/#tag/Posts/paths/~1posts.json/post)
+automatically from a script or tool, you can use the
+[Discourse API](https://docs.discourse.org/).
+
+### Who are the admins for Discourse?
+
+See https://llvm.discourse.group/about
+
+### What is the reason for the migration?
+
+see [this email](https://lists.llvm.org/pipermail/llvm-dev/2021-June/150823.html)
+
+### How do I set up a private mailing list?
+
+If needed categories can have individual [security
+settings](https://meta.discourse.org/t/how-to-use-category-security-settings-to-create-private-categories/87678)
+to limit visibility and write permissions. Contact the
+[admins](https://llvm.discourse.group/about) if you need such a category.
+
+### What will happen to our email archives?
+
+**TODO:** Explain the migration.
+
+**TODO:** Can we preserve the links to the email archive?
+see [#46](https://github.com/llvm/llvm-iwg/issues/46)
+
+### How Do I cross-post (send in multiple categories) a message?
+
+Topics are assigned to exactly one category, assigning to multiple categories
+[is not supported](https://meta.discourse.org/t/selecting-multiple-category/116827).
+
+**TODO:** The documentation says tags can be used to achieve a similar outcome.
+However they don't say how to do that.
```

Comment:
There are other ways to emulate cross-posting.  For example, you can have an 'announce' category or something that everyone is automatically subscribed too.  I think the important thing about cross posting is that it reaches people who may be subscribe only to a single category.  Where the archive of the discussion ends up is not all that important.

Reply:
I am not convinced that this is addressing "cross-posting" in general.
For example some technical discussion that starts on cfe-dev gets cross-posts to llvm-dev, but they aren't "announcements" neither they are suitable for a global category (irrelevant for lldb, lld, mlir, etc.).